cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -132,13 +132,10 @@ public Cube get() {
                                     ImmutableList.of(
       Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.COMPONENT)));
 
-    // component, handler, method, stream (for stream only) todo: seems like emitted context is wrong, review...
-    aggs.add(new DefaultAggregation(ImmutableList.of(
-      Constants.Metrics.Tag.NAMESPACE,
-      Constants.Metrics.Tag.COMPONENT, Constants.Metrics.Tag.HANDLER, Constants.Metrics.Tag.METHOD,
-      Constants.Metrics.Tag.STREAM),
-                                    // i.e. for stream only
-                                    ImmutableList.of(Constants.Metrics.Tag.STREAM)));
+    // stream
+    aggs.add(new DefaultAggregation(ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.STREAM),
+                                    // i.e. for streams only
+                                    ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.STREAM)));
```

Comment:
i see we emit metrics in the context, 
ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, Constants.SYSTEM_NAMESPACE,
                           Constants.Metrics.Tag.COMPONENT, Constants.Gateway.METRICS_CONTEXT,
                           Constants.Metrics.Tag.HANDLER, Constants.Gateway.STREAM_HANDLER_NAME,
                           Constants.Metrics.Tag.INSTANCE_ID, cConf.get(Constants.Stream.CONTAINER_INSTANCE_ID, "0")); before Constants.Metrics.Tag.STREAM in StreamHandler. 
though the first three tag-names are constants, do we need to include instance-id?